examples/pyramid_example.py

```diff
@@ -0,0 +1,83 @@
+"""A simple number and datetime addition JSON API.
+Run the app:
+
+    $ python examples/flask_example.py
+
+Try the following with httpie (a cURL-like utility, http://httpie.org):
+
+    $ pip install httpie
+    $ http GET :5001/
+    $ http GET :5001/ name==Ada
+    $ http POST :5001/add x=40 y=2
+    $ http POST :5001/dateadd value=1973-04-10 addend=63
+    $ http POST :5001/dateadd value=2014-10-23 addend=525600 unit=minutes
+"""
+
+import datetime as dt
+
+from wsgiref.simple_server import make_server
+from pyramid.config import Configurator
+from pyramid.view import view_config
+from pyramid.renderers import JSON
+from webargs import fields, validate
+from webargs.pyramidparser import use_args, use_kwargs
+
+
+hello_args = {
+    'name': fields.Str(missing='Friend')
+}
+
+
+# TODO: support
```

Comment:
Can you remove these and report issues for the missing features you are referring to?